### PR TITLE
Tribune to tts renaming

### PR DESCRIPTION
**Pull request by mradzisz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,37 @@
-# tribune-client
-Tribune TTS gRPC client
+# tts-client
```
Comment (YASoftwareDev) [inline README.md]: Uppercase

File: README.md [hunk 2]
```diff
@@ -1,39 +1,37 @@
...
+Techmo TTS gRPC client
 
 Contents:
```
Comment (YASoftwareDev) [inline README.md]: Contents: -> 
# Structure

File: README.md [hunk 3]
```diff
@@ -1,39 +1,37 @@
-# tribune-client
-Tribune TTS gRPC client
+# tts-client
+Techmo TTS gRPC client
```
Comment (YASoftwareDev) [inline README.md]: Techmo TTS gRPC client ->

This repository contains Techmo TTS gRPC client implementations.

File: README.md (hunk 1 above)
Comment (mradzisz) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (mradzisz) [reply README.md]: Fixed.

File: README.md (hunk 3 above)
Comment (mradzisz) [reply README.md]: Changed to `This repository contains Techmo TTS gRPC service API and client implementations.`

**Pull request merged**